js/merge_deps.bzl

```diff
@@ -0,0 +1,30 @@
+def merge_deps(ctx, tool_attr, dep_attr):
```

Comment:
I think this file belongs under private/

Reply:
rules authors will need this helper so I am not sure where it belongs. I have the feeling that it should be considered a public API